### ISSUE TITLE
Edit Groups Notifications

### ISSUE THREAD

**Issue by benterz222** [PHPfox-Official/phpfox-v4-feature-requests#945]
The ability to manage your notifications for Groups. Currently you are automatically subscribed to all notifications from within a group that you're a member of, which might annoy some users. So the ability to decide which notifications you would like to receive for the groups you're a member of.

**Comment by sryn101** [comment]
That's correct.. especially when someone posts images and videos, email notification is sent and there is no option to disable it.. there should be a smart way to send a weekly or daily digest of trending posts, images and videos in a group to all group members.
